### PR TITLE
Expose addok version

### LINKED ISSUE #268: Expose addok version
When running the "addok" bin, we should print the current addok version.
And I'm tempted to add this to the returned geojson too.

### PR DESCRIPTION
**Pull request by yohanboniface**
Fix #268

### CODE REVIEW

File: addok/http/base.py [hunk 1]
```diff
@@ -82,6 +83,7 @@ def to_geojson(self, req, resp, results, query=None, filters=None,
             "features": [r.format() for r in results],
             "attribution": config.ATTRIBUTION,
             "licence": config.LICENCE,
+            "addok": VERSION,
```
Comment (yohanboniface) [inline addok/http/base.py]: "version" is reserved for schema version (GeoCodeJSON version in theory), so I used this key instead. Any better idea?

Comment (davidbgk) [inline addok/http/base.py]: do we really need it within the payload?

File: addok/bin/__init__.py [hunk 2]
```diff
@@ -4,9 +4,12 @@
 import os
 
 from addok.config import config
+from addok import VERSION
```
Comment (davidbgk) [inline addok/bin/__init__.py]: Should be one line up?

File: addok/bin/__init__.py [hunk 3]
```diff
@@ -4,9 +4,12 @@
...
 
 
 def main():
+    if VERSION:
+        print('Addok', VERSION)
```
Comment (davidbgk) [inline addok/bin/__init__.py]: Any way to display plugins' versions too?

File: addok/http/base.py (hunk 1 above)
Comment (yohanboniface) [inline addok/http/base.py]: Not sure, no. Move back?

File: addok/bin/__init__.py (hunk 2 above)
Comment (yohanboniface) [inline addok/bin/__init__.py]: Fixed!

File: addok/http/base.py (hunk 1 above)
Comment (davidbgk) [inline addok/http/base.py]: I'm more interested in the version of the format that of the formatter so I would not add it to the payload.

That being said, we need to work on that `draft` thing :)

File: addok/bin/__init__.py (hunk 3 above)
Comment (yohanboniface) [inline addok/bin/__init__.py]: Added!

File: addok/config/__init__.py [hunk 4]
```diff
@@ -38,8 +42,17 @@ def load(self):
...
+        for name, plugin in hooks.plugins.items():
+            if name.startswith('addok.'):
+                version = VERSION  # Core plugin.
+            elif pkg_resources:
+                version = pkg_resources.get_distribution(
+                                plugin.__package__).version
+            else:
+                version = '?'
+            plugins.append('{}=={}'.format(name, version))
+        print('Loaded plugins:\n{}'.format(', '.join(plugins)))
```
Comment (davidbgk) [inline addok/config/__init__.py]: 👍

File: addok/http/base.py (hunk 1 above)
Comment (yohanboniface) [inline addok/http/base.py]: > I'm more interested in the version of the format that of the formatter so I would not add it to the payload.

We still could have both, they have different meaning ;)
But okay, dropped the commit!

**Pull request merged**